src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -284,6 +286,10 @@ func (s *cloudService) CreateVM(ctx context.Context, req *pb.CreateVMRequest) (r
 		}
 	}
 
+	// Set scratch disk configuration
+	daemonConfig.EnableScratchDisk = s.serverConfig.EnableScratchDisk
+	daemonConfig.EnableScratchEncryption = s.serverConfig.EnableScratchEncryption
```

Comment:
do we need a check to confirm that scratch encryption can only be set if scratch is enabled?

Reply:
I think it's better to check it early. I'll add it in the next iteration